### DISCUSSION TITLE
Dap example?

### DISCUSSION THREAD

**Discussion by aleyrizvi** [nvim-java/nvim-java#217] [Q&A]
Hello,
I am new to neovim and java all together (yeah right :D) 

I tried nvim-jdtls and made it working with dap. I am using lazyvim starter. 

It did not have lombok support and since I am new, I couldn't configure it.

I tried nvim-java and it worked flawlessly, only that...the debugger is not working.

It says no config found.

Can you help please?

**Comment by s1n7ax** [comment]
@aleyrizvi First you have to get rid of anything existing related to java from the configuration. Then follow the instructions in the readme. Sometimes debugger is not working because it cannot find the project. To make sure, you can create a directory, and make it a git repository. Then create a java file and try to debug.

**Comment by s1n7ax** [comment]
https://github.com/nvim-java/play-lazyvim

This is a sample configuration on lazyvim with nvim-java.

**Comment by s1n7ax** [comment]
@aleyrizvi I just realized even my local env is broken. It looks like due to the recent lazy.nvim changes. I'm looking into it right now.

**Comment by s1n7ax** [comment]
@aleyrizvi Do you have your config in github?

**Reply by aleyrizvi** [reply]
Thanks
Here is the link: 

https://github.com/aleyrizvi/nvim-config

**Comment by s1n7ax** [comment]
@aleyrizvi 
I have changed the config for LazyVim. Please update that first. Do a full plugin update to get the latest LazyVim.

https://github.com/nvim-java/nvim-java/wiki/Lazyvim

For debugger, you have to add the LazyExtra module like below.
https://github.com/s1n7ax/lazyvim-dotnvim/blob/6a20213a74bf58445c82efee40bc63e79364f623/lazyvim.json#L3

Lombok should be working by default.

**Comment by s1n7ax** [comment]
@aleyrizvi Note that you should not enable java in lazyvim or install nvim-jdtls for this plugin to work
